zanata-war/pom.xml

```diff
@@ -163,6 +161,7 @@
         <groupId>com.ning.maven.plugins</groupId>
         <artifactId>maven-duplicate-finder-plugin</artifactId>
         <configuration>
+          <checkTestClasspath>false</checkTestClasspath>
```

Comment:
The danger of tests not testing what we think is too high a price to pay for a (possibly) smaller download on the client.